### ISSUE TITLE
Tiled window is unclickable sometimes after session unlock

### ISSUE THREAD

**Issue by develop7** [paperwm/PaperWM#758]
**Describe the bug**
Title

**To Reproduce**
1. Lock GNOME session, explicitly or otherwise
2. Unlock the session
3. Try clicking the active window with mouse

**Expected behavior**
Well, the window gets clicked, as it is normally.

**What happens instead**
The active window doesn't react to clicks (not sure of other events, will update) until being reactivated by switching to some other window back & forth. The issue sometimes repeats in other workspaces (gets fixed same way).

**Screenshots**
Not yet

**System information:**
Please execute `./gather-system-info.sh` in you PaperWM clone and paste the output below.

```
Distribution: openSUSE Tumbleweed
GNOME Shell 45.3
Display server: Xorg
PaperWM branch/tag: release
PaperWM commit: e2de6b168af5e258359ef7099f7b2f3f31b78285
Enabled extensions:
- user-theme@gnome-shell-extensions.gcampax.github.com
- expandable-notifications@kaan.g.inam.org
- middleclickclose@paolo.tranquilli.gmail.com
- weatheroclock@CleoMenezesJr.github.io
- osd-volume-number@deminder
- gsconnect@andyholmes.github.io
- just-perfection-desktop@just-perfection
- solaar-extension@sidevesh
- tophat@fflewddur.github.io
- window-title-is-back@fthx
- paperwm@paperwm.github.com
- panelScroll@sun.wxg@gmail.com
- hotedge@jonathan.jdoda.ca
- appindicatorsupport@rgcjonas.gmail.com
- bluetooth-quick-connect@bjarosze.gmail.com
```

**Additional context**

**Comment by jtaala** [comment]
Hey @develop7, this bug should be resolved once we get `develop` merged into release.

I think It's caused by the `ClickOverlay` element (which was replaced in `develop`).  Checkout `develop` branch and give that a go.

Note: there's still some touchscreen support issues in develop that we're working through.

**Comment by jtaala** [comment]
Scrap that - happens on `develop` too.

@develop7 - I can reproduce this for windows that autostart (i.e. start on gnome startup, and often before PaperWM starts) - is that the case with your setup?

It might happen for other windows, but I just noticed it for `brave` (browser) windows (I autostart brave).

**Comment by Lythenas** [comment]
I think I remember I also saw this in Thunderbird and Firefox in the past. Although not for a while. I think since they had that big update sometime last year. Also I think this was the reason I stopped autostarting Firefox^^

**Comment by jtaala** [comment]
See #760 which fixes this.  I'm not sure if this is a "autostart" windows only issue (or occurs when we have autostarted windows) - but #760 fixes @develop7 particular issue.

**Comment by jtaala** [comment]
Re-opening this until updated PR is merged.  #760 fixed this for the active space only (should be applied for all spaces on init).

**Comment by jtaala** [comment]
Fixed with #761
